chapter_multilayer-perceptrons/mlp_vn.md

```diff
@@ -276,7 +277,11 @@ That means that after computing the linear portion of the layer, we can calculat
 This is true for most activation functions (the batch normalization operation will be introduced in :numref:`sec_batch_norm` is a notable exception to that rule).
 -->
 
-*dịch đoạn phía trên*
+Khác với các quy ước thông thường, chúng tôi định nghĩa phi tuyến $\sigma$ áp dụng cho các giá trị đầu vào theo từng dòng, tức là mỗi một điểm dữ liệu quan sát được tại một thời điểm.
```

Comment:
```suggestion
Khác với các quy ước thông thường, chúng ta định nghĩa phi tuyến $\sigma$ áp dụng cho các giá trị đầu vào theo từng dòng, tức mỗi điểm dữ liệu chỉ quan sát được tại một thời điểm.
```

Reply:
Mình thêm từ `hàm` vào trước `phi tuyến` trong suggestion của @duythanhvn.

Reply:
Em thắc mắc chỗ này xíu
Đoạn văn gốc là "apply to its inputs in a row-wise fashion, i.e., one observation at a time"
Theo em observation là dữ liệu quan sát được, nên nếu dịch thành "mỗi điểm dữ liệu chỉ quan sát được tại một thời điểm" có vẻ không đúng lắm?
"chỉ mỗi điểm dữ liệu quan sát được tại một thời điểm" nghe có vẻ đúng hơn?

Reply:
mình sẽ dịch như thế này
```suggestion
Khác với các quy ước thông thường, chúng ta định nghĩa hàm phi tuyến $\sigma$ áp dụng cho các giá trị đầu vào theo từng hàng, tức lần lượt từng điểm dữ liệu một.
```

Reply:
ok